src/utils/stringifiers.ts

```diff
@@ -0,0 +1,61 @@
+import type { BasicType, Type } from '../interfaces.js';
+
+export function defaultStringify(valueType: BasicType | 'mixed', value: unknown, name: string): string | undefined {
+    switch (valueType) {
+        case 'function':
+        case 'symbol':
+        case 'undefined':
+            return undefined;
+        case 'bigint':
+            return fallbackStringify(value);
```

Comment:
`fallback` to let `JSON.stringify` throw the error? Of in case it gets support in the future? 🤔  Either way, could use a comment. :-)

Reply:
No, because people can do crazy things:
<img width="455" alt="image" src="https://github.com/skunkteam/types/assets/5168053/c475f04b-6978-48eb-a0c0-290f3abb7206">